qek/data/graphs.py

```diff
@@ -519,7 +519,7 @@ class NXGraphCompiler(BaseGraphCompiler[NXWithPos]):
     def ingest(self, graph: NXWithPos, device: pl.devices.Device, id: int) -> BaseGraph:
         pyg = pyg_utils.from_networkx(graph.graph)
         pyg.y = graph.target
-        positions = [graph.positions[node] for node in graph.graph.nodes()]
+        positions = np.array([graph.positions[node] for node in graph.graph.nodes()])
```

Comment:
Can't you simply:

```python
pyg.pos = torch.tensor([graph.positions[node] for node in graph.graph.nodes()], dtype=torch.float)
```

Reply:
Yes, I can. I felt that it was slightly more readable in two lines instead of one.

Reply:
Well, this would trigger the exact same warning.